### ISSUE TITLE
Thanks

### ISSUE THREAD

**Issue by devildant** [litinoveweedle/hass-template-climate#10]
Hello,

just a thank you message, I was afraid to see this addon disappear like many no longer maintained projects.

and thank you for retrieving and merging my pull request :)

(I already added a star :p)

**Comment by litinoveweedle** [comment]
Hello @devildant, you are welcome, and big thanks you for your original PR :1st_place_medal:. Without it I would never start with this fork . It is BTW beauty of FOSS, as we can all help each to other. And thank you for the star!

**Comment by s1mpleman** [comment]
Rather than opening a new thread I'd like to add my thanks as well.

What you've done expanding this is wonderful. I'd just started writing up a macro for my preset values to keep them in a single place before I saw your comment on the original repository. 

Now with this I've replaced Versatile Thermostat, a myriad of Automations/scripts and Wiser TRV overrides. All my occupied heating is just a template now and much easier to manage and it successfully integrates over the top of my wiser climate devices supporting the passive mode feature and other stuff I've had to write complication automations to support in the past. 

Thank you!
